<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_send">
        <property name="text">
         <string>Mesaj Yolla</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_disconnect">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>Sunucuyu kapat</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEdit_datasize"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_connect">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>Sunucuya bağlan</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Toplam Veri(byte)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLineEdit" name="lineEdit_message"/>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="lineEdit_packagesize"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_speed"/>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="pushButton_teststart">
        <property name="text">
         <string>Testi Başlat</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="pushButton_compare">
        <property name="text">
         <string>Log Kıyasla</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="7">
       <widget class="QTextEdit" name="textEdit_log"/>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Paket(byte)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Hız(kb/sn)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_status">
        <property name="maximumSize">
         <size>
          <width>9999</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>Durum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
